### DISCUSSION TITLE
i need help

### DISCUSSION THREAD

**Discussion by ghost** [community/community#48367] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

uncle lembani i need your help with understanding github

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @Andresakala  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Reply by ghost** [reply]
thanks
